### DISCUSSION TITLE
A future about versioning in minio.

### DISCUSSION THREAD

**Discussion by OlegPyzin** [minio/minio#16946] [Show and tell]
Hello Dear Developers!
In my job I am using a lot minio services.
Thank you incredible job.
But I would like to write about some things about versioning in the minio service.
Before I started work with MINIO I am working a lot on OpenVMS operating system as admin and as a programmer.

It was great to expand the capabilities of the service for storing versions of a single file to the capabilities that are in the OpenVMS operating system.

If the developer community allows me to join this work, I will gladly join you.

With the Best Regards,
Oleg

**Comment by klauspost** [comment]
@OlegPyzin Thanks!

I think the reason you haven't got a response is that you don't really ask a question, and it is rather unclear what you'd like to do.

If you would like to write an article (blog post), you are very welcome to write `hello@min.io` with an outline and we can see if it is a good fit.
